submissions/Oleksandr-Heleta/popup/index.html

```diff
@@ -0,0 +1,168 @@
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/files.png" alt="">
+                                    <p>Failes</p>
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/contacts.png" alt="">
+                                    <p>Contacts</p>
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/photos.png" alt="">
+                                    <p>Photos</p>
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/apps.png" alt="">
+                                    <p>Apps</p>
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/tanslation.png" alt="">
+                                    <p>Translation</p>
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/blogs.png" alt="">
+                                    <p>Blog</p>
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/videos.png" alt="">
+                                    <p>Video</p>
+                                </a>
+                            </li>
+                            <li class="popup_item">
+                                <a class="popup_link" href="#" target="_blank" rel="noopener noreferrer">
+                                    <img class="popup_img" src="./popup-icons/search.png" alt="">
```

Comment:
Please fix the vertical alignment of the icons 
![image](https://user-images.githubusercontent.com/16196199/185802752-0be3c913-a8c0-43ac-87a4-27633c741733.png)

Reply:
I used flex-end. I hope it will be better). Please review one more